yogstation/code/modules/lighting/lighting_turf.dm

```diff
@@ -0,0 +1,18 @@
+/turf/proc/get_rgb_lumcount(var/minlum = 0, var/maxlum = 1, var/r_mul = 1, var/g_mul = 1, var/b_mul = 1)
```

Comment:
Can't you use ``/turf/proc/get_lumcount``? Look at the nightmare code.

Reply:
this is really similar but multiplies it so RGB affects it and shit, best explanation I can do. can probably merge it with the nightmare one